bundle/manifests/lvm-operator.clusterserviceversion.yaml

```diff
@@ -17,8 +17,8 @@ metadata:
                   "name": "vg1",
                   "thinPoolConfig": {
                     "name": "thin-pool-1",
-                    "overprovisionRatio": 50,
-                    "sizePercent": 50
+                    "overprovisionRatio": 10,
+                    "sizePercent": 90
```

Comment:
Let's set the sizePercent to 75% so as not to trigger the VG utilization alerts.

Reply:
We do have vg utilization alert set at 75% as warning and 85% as critical.

Reply:
The alert is sent once it crosses 75%. Would setting it to exactly 75% trigger it?
The other option is to change the alert to fire at 80% and 85%.

Reply:
so keeping sizePercent as 90 like we discussed in the planning meeting. I'll open another PR if we decide to remove/change the Volume Group alerts.